### PR TITLE
v2.0 release 

### PR DESCRIPTION
**Pull request by siduck**
https://github.com/orgs/NvChad/projects/8

Other pr's related to this release

https://github.com/NvChad/ui/pull/51
https://github.com/NvChad/base46/pull/156

Rewrite whole site in solidjs + vite, remove docusaurus cuz its a lil slow and bloated

https://github.com/NvChad/nvchad.github.io/pull/118

Edit : v2.0 has released and this PR will merge after 3-4 months when users have migrated to v2.0 properly

### CODE REVIEW

**Comment by ynhhoJ** [comment]
When is the CHAD release coming out? 👀👀👀

**Comment by siduck** [comment]
> When is the CHAD release coming out? eyeseyeseyes

within next 2 weeks most likely, only thing to now do is finish writing the site and probably i'll do some presentations on youtube :nerd_face:

**Comment by cliffordfajardo** [comment]
Just saw this Repo on my newsfeed; been wanting to get into Neovim for a while; glad to try this out as a first time neovim user!

File: .github/README.md [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 
 <div align="center">
  
-[![Super Linter](https://img.shields.io/github/workflow/status/NvChad/NvChad/Super-Linter/main?style=flat-square&logo=github&label=Build&color=8DBBE9)]()
-<a href="https://github.com/NvChad/NvChad/blob/main/LICENSE"
-        ><img
-            src="https://img.shields.io/github/license/NvChad/NvChad?style=flat-square&logo=GNU&label=License&color=df967f"
-            alt="License"
-    />
 [![Neovim Minimum Version](https://img.shields.io/badge/Neovim-0.8.0-blueviolet.svg?style=flat-square&logo=Neovim&color=90E59A&logoColor=white)](https://github.com/neovim/neovim)
```
Comment (mikededo) [inline .github/README.md:18]: Docs say minimum version is `0.8.3`. Which one should stay?

Comment (siduck) [reply .github/README.md:18]: @mikededo need to update the readme, thx for correcting!

Comment (mikededo) [reply .github/README.md]: I was looking through the docs today and I found a couple of improvements, do you want me to later create a PR with it? I would be willing to help with the docs 💪

Comment (siduck) [reply .github/README.md]: @mikededo feel free to!  Btw do know that all the docs are in md files, `/docs and /news` format.

**Review by Just-Moh-it — APPROVED** [review]

**Comment by siduck** [comment]
I just realized we dont need this PR, all versions of nvchad will stay in their own branches!

**Pull request closed without merging**